### PR TITLE
Add support for Django Password Validators

### PR DESCRIPTION
**Pull request by piotr-szpetkowski**
Remove support for DRF validators as password validators
Update documentation
Update tests

Based on #176 and #177

### CODE REVIEW

**Comment by haxoza** [comment]
See #147

File: testproject/testapp/validators.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-def is_666(value):
-    from rest_framework import serializers
-    if value == '666':
-        raise serializers.ValidationError('Woops, 666 is not allowed.')
+from django.core.exceptions import ValidationError
+
+
+class Is665(object):
```
Comment (KaczuH) [inline testproject/testapp/validators.py]: looks like typo. `Is665` or `Is666`?

Comment (piotr-szpetkowski) [inline testproject/testapp/validators.py]: I've pulled it from #176 so other teammate might get a credit for contribution (I'll probably split the PR before merge and blame her).

**Comment by haxoza** [comment]
Great job! Happy to see you're reused @agagata tests! 🏆

**Pull request merged**